tests/drum/test_unstructured_mode.py

```diff
@@ -227,25 +228,37 @@ def test_response_one_var_return(
             data = None
             headers = {"Content-Type": "text/plain; charset=UTF-8"}
             response = requests.post(url=url, data=data, params=params, headers=headers)
-            content_type_header = response.headers["Content-Type"]
+
```

Comment:
you can also add tests to
1. send text data with no charset
2. send text data with non-utf-8 charset (since utf-8 is a default one)
3. send binary data with encoding

Reply:
Added more test cases, not sure all you mention. :)